desc/objectives/linear_objectives.py

```diff
@@ -2611,6 +2611,85 @@ def __init__(
         )
 
 
+class FixCoilCurrent(FixParameters):
+    """Fixes current(s) in a Coil or CoilSet.
+
+    Parameters
+    ----------
+    coil : Coil
+        Coil(s) that will be optimized to satisfy the Objective.
+    target : {float, ndarray}, optional
+        Target value(s) of the objective. Only used if bounds is None.
+        Must be broadcastable to Objective.dim_f. Default is ``target=eq.Psi``.
+    bounds : tuple of {float, ndarray}, optional
+        Lower and upper bounds on the objective. Overrides target.
+        Both bounds must be broadcastable to to Objective.dim_f.
+        Default is ``target=eq.Psi``.
+    weight : {float, ndarray}, optional
+        Weighting to apply to the Objective, relative to other Objectives.
+        Must be broadcastable to to Objective.dim_f
+    normalize : bool, optional
+        Whether to compute the error in physical units or non-dimensionalize.
+    normalize_target : bool, optional
+        Whether target and bounds should be normalized before comparing to computed
+        values. If `normalize` is `True` and the target is in physical units,
+        this should also be set to True.
+    indices : nested list of bool, optional
```

Comment:
I think it would be good to include an example of this (can basically just use the one from the test). Also the part about `Must have the same pytree structure as coil.params_dict.` isn't quite right I think, because its a list of bools, not a list of dict of bool? Not sure the best way to describe that, so I think an example would be clearest.